### PR TITLE
fix(llvm-git): Fix the build and pkgver

### PR DESCRIPTION
**Pull request by SolarAquarion**
it wasn't building and now it's building.  I also moved things to the source

### CODE REVIEW

File: archlinuxcn/llvm-git/PKGBUILD [hunk 1]
```diff
@@ -53,9 +58,7 @@ prepare() {
     # llvm-project contains a lot of stuff, remove parts that aren't used by this package
     rm -rf "$srcdir"/fakeinstall
 
-  cd llvm/projects
-  git clone https://github.com/KhronosGroup/SPIRV-LLVM-Translator.git
-
+  cp -r $srcdir/SPIRV-LLVM-Translator llvm/projects
 }
 
 build() {
```
Comment (Zenvie) [inline archlinuxcn/llvm-git/PKGBUILD:64]:
```
==> ERROR: A failure occurred in build().
```
https://fars.ee/Nclk.log

Comment (SolarAquarion) [reply archlinuxcn/llvm-git/PKGBUILD:64]: the issue is that there isn't enough storage on your device.

Comment (SolarAquarion) [reply archlinuxcn/llvm-git/PKGBUILD:64]: "/build/llvm-git/src/_build/bin/llvm-exegesis" to
  "/build/llvm-git/src/fakeinstall/usr/bin/llvm-exegesis": No space left on
  device.

Comment (Zenvie) [reply archlinuxcn/llvm-git/PKGBUILD:64]: I'm using a build server provided by the Arch Linux CN community, so insufficient storage space is unlikely.

Comment (SolarAquarion) [reply archlinuxcn/llvm-git/PKGBUILD:64]: so the size of the build server needs to be expanded

Comment (lilydjwg) [reply archlinuxcn/llvm-git/PKGBUILD:64]: @Zenvie you ran out the tmpfs space for the chroot by default. lilac has been instructed to `-r` to choose another place for it.

Comment (oldherl) [reply archlinuxcn/llvm-git/PKGBUILD:64]: @lilydjwg Is there an option in lilac.yaml that can instruct lilac to build the package outside tmpfs?

Comment (Zenvie) [reply archlinuxcn/llvm-git/PKGBUILD:64]: > @Zenvie you ran out the tmpfs space for the chroot by default. lilac has been instructed to use `-r` to choose another place for it.

This parameter has the unintended consequence of creating files that I lack the necessary permissions to delete.

**Comment by SolarAquarion** [comment]
you couldn't delete those files before.  But I just moved it from a dirty
method to a clean method.  I don't think there should be a git clone within
git.  You would be having the same issue if it was a git submodule

File: archlinuxcn/llvm-git/PKGBUILD (hunk 1 above)
Comment (lilydjwg) [reply archlinuxcn/llvm-git/PKGBUILD:64]: @oldherl it's in [lilac.py](https://github.com/archlinuxcn/lilac/blob/master/schema-docs/lilac-py-fields.md).

@Zenvie yes....

**Pull request merged**